src/storage/src/hummock/sstable_store.rs

```diff
@@ -655,7 +655,7 @@ impl SstableStore {
             _ => (),
         }
         stats.cache_meta_block_total += 1;
-        lookup_response.verbose_instrument_await("sstable")
+        lookup_response
```

Comment:
Is there any indication that `await-tree` results in high CPU overhead?

Reply:
Yes. A little... we tested it but i can not found the flamegraph now ...

Reply:
I believe there's always a trade-off between observability and performance. 🤣 If we find the impact little, perhaps we should keep it untouched.